blocks/feed/feed.js

```diff
@@ -0,0 +1,51 @@
+import {
+  buildBlock, decorateBlock, loadBlock, readBlockConfig,
+} from '../../scripts/lib-franklin.js';
+import { queryIndex } from '../../scripts/scripts.js';
+
+const resultParsers = {
+  cards: (results) => {
+    const blockContents = [];
+    results.forEach((result) => {
+      const resultTitle = document.createElement('div');
```

Comment:
Also suppose we have to add custom fields to display for some specific fields lets suppose date, then i think that should also be derived on the basis of feed type.

Reply:
Good idea. Added.